certified-connectors/Payables E2E/apiDefinition.swagger.json

```diff
@@ -0,0 +1,132 @@
+    "description": "A smart invoice processing solution that converts static documents (scanned, PDFs, screenshots, etc.) into usable information to fuel your business processes.IN-D Payables is an AI led intelligent document classification and capture solution that automates processing of all kinds of commercial documents used in Shipping, Logistics and Manufacturing operations. Manual handling of documents slows the pace of business transactions."
+  },
+  "host": "payables-sit.in-d.ai",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/users/token/": {},
+    "/api/e2e": {
+      "post": {
+        "summary": "Payable E2E",
+        "description": "Process invoices to extract the key attributes like invoice numbers, PO number, vendor name, etc., and export the data",
+        "operationId": "Payable_E2E",
+        "parameters": [
+          {
+            "name": "Accept-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "v3.0",
+            "description": "Accept-Version"
+          },
+          {
+            "name": "Accept-Export-Type",
+            "in": "header",
+            "required": true,
+            "type": "integer",
+            "description": "Accept-Export-Type",
+            "default": 1
+          },
+          {
+            "name": "Accept-Attachment",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "True",
+            "description": "Accept-Attachment"
+          },
+          {
+            "name": "Accept-Route",
+            "in": "header",
+            "required": true,
+            "type": "integer",
+            "default": 1,
+            "description": "Accept-Route"
+          },
+          {
+            "name": "Accept-Doctype",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "1,2",
+            "description": "Accept-Doctype"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "files": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "files",
+                    "title": "Base64 format"
+                  },
+                  "description": "files"
+                }
+              },
+              "default": {
+                "files": [
+                  "/9j/4AAQSkZJ...(Base64 value)"
+                ]
+              },
+              "required": [
+                "files"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@gayathri35 default responses with an empty schema are [not allowed](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses), as well as using only default schema for responses. Please define possible responses for your operations. You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).